File: Lógica/Lógic C#/POO/Att11/Models/Desenvolvedor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Att11.Models;

namespace Att11.Models {
    public class Desenvolvedor : Funcionario {
        public Desenvolvedor(string nome, decimal salario) : base(nome, salario) { }
       
        public override decimal CalcularSalarioLiquido(){
            return Salario * 1.10m;
        }
    }
}

File: Lógica/Lógic Jav/src/POO/att16/src/Models/Desenvolvedor.java
package att16.src.Models;

import att16.src.Enums.Cargos;

public class Desenvolvedor extends Funcionario{
    public Desenvolvedor(String nome, double salario) {
        super(nome, salario);
        this.cargo = Cargos.DESENVOLVEDOR;
    }
}
